src/lua/LuaShip.cpp

```diff
@@ -1146,7 +1146,7 @@ static int l_ship_get_gun_temperature(lua_State *l)
 {
 	Ship *s = LuaObject<Ship>::CheckFromLua(1);
 	int gun = luaL_checkinteger(l, 2);
-	LuaPush(l, s->GetGunTemperature(gun));
+	LuaPush(l, s->GetFixedGuns()->GetGunTemperature(gun));
```

Comment:
why not `GetComponent` ?

Reply:
I was too lazy to add a commit to change this specific occurrence - will clean that up.